### PR TITLE
gh-99 Add custom scrollbar and contributors to README

### PR DESCRIPTION
**Pull request by aditya-singh9**
I have added custom scrollbar in the webpage and contributions section in the readme.

### CODE REVIEW

File: packages/webapp-next/yarn.lock [hunk 1]
```diff
@@ -1,2601 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (codicocodes) [inline packages/webapp-next/yarn.lock:1]: Please do not remove the yarn lock, it should be persisted in the repository.

**Review by codicocodes — CHANGES_REQUESTED** [review]
Thank you for the contribution! I will look through and test this more later! 

But first of all the yarn.lock file should not be deleted!

File: packages/webapp-next/yarn.lock (hunk 1 above)
Comment (aditya-singh9) [reply packages/webapp-next/yarn.lock:1]: Okay i will add this file

Comment (aditya-singh9) [reply packages/webapp-next/yarn.lock]: Resolved

**Review by codicocodes — APPROVED** [review]

**Pull request merged**